bundles/org.openhab.ui/web/src/pages/settings/rules/script/blockly-editor.vue

```diff
@@ -328,156 +266,388 @@
+          <block type="oh_ephemeris_basic" />
+          <block type="oh_ephemeris_offset">
+            <value name="offset">
+              <shadow type="math_number">
+                <field name="NUM">
+                  0
+                </field>
+              </shadow>
+            </value>
+          </block>
+        </category>
+        <category name="Voice/Multimedia">
+          <block type="oh_playmedia_sink">
+            <value name="sinkName">
+              <shadow type="oh_audiosink_dropdown" />
+            </value>
+          </block>
+          <block type="oh_playmedia_sink_volume">
+            <value name="sinkName">
+              <shadow type="oh_audiosink_dropdown" />
+            </value>
+            <value name="volume">
+              <shadow type="audioSlider">
+                <field name="volume">5</field>
+              </shadow>
+            </value>
+          </block>
+          <block type="oh_playstream_sink">
+            <value name="sinkName">
+              <shadow type="oh_audiosink_dropdown" />
+            </value>
+          </block>
+          <block type="oh_stopstream_sink">
+            <value name="sinkName">
+              <shadow type="oh_audiosink_dropdown" />
+            </value>
+          </block>
+          <block type="oh_say">
+            <value name="textToSay">
+              <shadow type="text">
+                <field name="TEXT">text to say</field>
+              </shadow>
+            </value>
+            <value name="deviceSink">
+              <shadow type="oh_audiosink_dropdown" />
+            </value>
+            <value name="voice">
+              <shadow type="oh_voices_dropdown" />
+            </value>
+          </block>
+        </category>
+        <category name="Logging/Output">
+          <block type="oh_log">
+            <value name="message">
+              <shadow type="text">
+                <field name="TEXT">abc</field>
+              </shadow>
+            </value>
+          </block>
+          <block type="oh_print">
+            <value name="message">
+              <shadow type="text">
+                <field name="TEXT">abc</field>
+              </shadow>
+            </value>
+          </block>
+        </category>
       </category>
-
       <sep />
-      <category name="Variables" colour="%{BKY_VARIABLES_HUE}" custom="VARIABLE" />
-      <category name="Functions" colour="%{BKY_PROCEDURES_HUE}" custom="PROCEDURE" />
+      <category
+        name="Variables"
+        colour="%{BKY_VARIABLES_HUE}"
+        custom="VARIABLE" />
+      <category
+        name="Functions"
+        colour="%{BKY_PROCEDURES_HUE}"
+        custom="PROCEDURE" />
     </xml>
   </div>
 </template>
 
 <style lang="stylus">
-.blockly-editor
-  display block
-  top calc(100%)
-  height calc(100%)
-  width 100%
-  .blockly
-    height 100%
-    .blocklyMainBackground
-      stroke inherit
-.blocklyDropDownDiv
-  z-index 9000
+.blockly-editor {
```

Comment:
Please revert back to the Stylus syntax.

Reply:
I mean the style changes without the {} and : and ; where perfectly fine as they were. Stylus is a simplified CSS syntax.